Extending/index-v8.md

```diff
@@ -8,6 +8,9 @@ versionFrom: 8.0.0
 These features are only available in v8
 :::
 
-## [Content Apps](Content-Apps/index-v8)
+## [Content Apps](Content-Apps/index-v8.md)
```

Comment:
Adding .md here is not necessary as the link manager looks for both folders and files with the name, if you test it out on the current page the link works just fine 😉

Reply:
I am merging this, as it's a file anyway.  The .md extension makes it work on github too...